pandas/tests/series/test_constructors.py

```diff
@@ -966,7 +966,8 @@ def test_constructor_dtype_datetime64_10(self):
         # GH3414 related
         expected = Series(pydates, dtype="datetime64[ms]")
 
-        result = Series(Series(dates).view(np.int64) / 1000000, dtype="M8[ms]")
+        with tm.assert_produces_warning(FutureWarning, match="view is deprecated"):
+            result = Series(Series(dates).view(np.int64) / 1000000, dtype="M8[ms]")
```

Comment:
```suggestion
        result = Series(Series(dates).astype(np.int64) / 1000000, dtype="M8[ms]")
```

(this test is not about `view()` but about the constructor from ints, so I think we can already update it and then we don't have to change it later again)